spot-client/src/common/backend/index.js

```diff
@@ -6,3 +8,6 @@ export {
     getRemotePairingCode,
     refreshAccessToken
 } from './utils';
+
+import './middleware';
+import './reducer';
```

Comment:
Are you sure we want to now have state in common/backend? Do we need it?

Reply:
We can have it either here or in "spot remote/backend" and "spot tv/backend". I though to keep it in one/common place as it turned out that it's the same thing for both TV and remote.

Reply:
My only concern is common/backend having knowledge of redux. If it's necessary for it to do so now then that's what has to happen. Otherwise I would like to keep a separation of redux and the backend logic if, and only if, reasonable.

Reply:
Well, really backend service class is not directly involved in redux, but Im thinking of this package as a feature like we have in JitsiMeet. I think it belongs here or I have no better idea. Let me know if you do.